examples/sw/benchmarks/README.md

```diff
@@ -11,9 +11,12 @@ All of these benchmarks run on Simple System. A verilator simulation suitable
 for running them can be built with:
 
 ```
-fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ibex:ibex_simple_system --RV32E=0 --RV32M=ibex_pkg::RV32MFast
+fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ibex:ibex_simple_system `./util/ibex_config.py [ibex_config] fusesoc_opts`
```

Comment:
Hmm, maybe it's cleaner to give an example config here? (That way a reader can copy-paste from the doc to their terminal!)

Reply:
So I switched this to specify 'maxperf' as the config so it can be copy/pasted. I didn't want to include explicit flags as the idea is you should just use the ibex_config.py script to generate them for you based upon a config name.

Some may wish to specify manually but I'd consider that a more advanced usage and these instructions are targetted for people just trying to get Ibex doing a few things, such as reproduce benchmark results against supported configs.

Reply:
@rswarbrick does that address your question?

Reply:
Oh, I see. Yep, that sounds reasonable to me.

Reply:
Aargh! Sorry, I forgot! I'd suggest picking a config here, and writing something below about how to change it. It seems importantant (to me) that a reader can just copy/paste *something* into the terminal.

Reply:
The example command line can be directly copy/pasted and used.

Though the wording below maybe didn't make this obvious, I've updated it to make it explicitly (hopefully!).

Reply:
Duh, sorry. I was looking at the thing from GitHub at the top of this discussion, which is what I originally commented on. What you've got looks good to me.